src/main/java/io/lettuce/core/ClientListArgs.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2011-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.lettuce.core;
+
+import io.lettuce.core.internal.LettuceAssert;
+import io.lettuce.core.protocol.CommandArgs;
+import io.lettuce.core.protocol.CommandType;
+
+import static io.lettuce.core.protocol.CommandKeyword.ID;
+
+/**
+ *
+ * Argument list builder for the Redis <a href="https://redis.io/commands/client-list">CLIENT LIST</a> command.
+ * <p>
+ * {@link ClientListArgs} is a mutable object and instances should be used only once to avoid shared mutable state.
+ *
+ * @author Mikhael Sokolov
+ * @since 6.3
+ */
+public class ClientListArgs implements CompositeArgument {
+
+    private enum Type {
+        NORMAL, MASTER, SLAVE, PUBSUB
+    }
+
+    private Long id;
```

Comment:
Seems that `CLIENT LIST` can accept multiple ID's

Reply:
Ah, indeed, thanks! Fixed